internal/cli/profile.go

```diff
@@ -133,6 +135,15 @@ func (p *Profile) resolveFlags() error {
 	}
 	p.SetString(keyTelemetryMode, string(p.telemetryMode))
 
+	if p.telemetryWriteKey == "" {
```

Comment:
all of this write key stuff should not be part of the profile, it should not be configurable by the user (they should only be able to flip the telemetry mode here)